### ISSUE TITLE
ini file - logdir param doesn't work

### ISSUE THREAD

**Issue by mcappuccioAB** [munin-monitoring/munin-node-win32#86]
Hello,
I'm trying to use munin-node on a Windows server, everything looks ok at moment, but "logdir" parameter doesn't work
It is because it use only Windows Event Log?
There is not a way to use both log systems?

Looking the source code, I cannot find where is the class who write on text log file

Thanks for reply
Mauro

**Comment by lemontree23** [comment]
Yes, at the moment it only uses the windows event log. You can always create a pull request with a new logging system implemented.
